roofit/roofitcore/inc/LinkDef.h

```diff
@@ -124,7 +124,6 @@
 #pragma link C++ class RooGenContext+ ;
 #pragma link C++ class RooGenericPdf+ ;
 #pragma link C++ class RooGenProdProj+ ;
-#pragma link C++ class RooHistError+ ;
```

Comment:
Why are you removing RooHistError from the LinkDef.h file ?

Reply:
The LinkDef entry is necessary for IO, but other than that the advantage of generating dictionaries is - as far as I know - only a little bit faster PyROOT when using this class because the dictionaries don't have to be generated on the fly.

However, this `RooHistError` is only an implementation detail on other RooFit classes and not meant to be used by the user. So I don't think the overhead of generating the dictionaries is worth it.

For the release notes, yes I will add something there!